tests/unit/test_run_logprep.py

```diff
@@ -162,6 +161,12 @@ def test_version_arg_prints_with_http_config(self, capsys):
 
     @responses.activate
     def test_version_arg_prints_with_http_config_without_exposing_secret_data(self, capsys):
+        os.environ.update(
```

Comment:
Can we maybe use a mock for the environment variables in tests? Like this for example:
```python
mock_env = {
    "LOGPREP_CONFIG_ATUH_USERNAME": "username",
    "LOGPREP_CONFIG_ATUH_PASSWORD": "password",
}
with mock.patch("os.environ", mock_env):
    <do stuff with env>
```

I think that is a somewhat cleaner implementation and we don't risk forgetting to delete the env changes again.

Reply:
done